modules/tests/test/copy_template_test.go

```diff
@@ -79,67 +77,52 @@ var _ = Describe("Copy template task", func() {
 			r := runner.NewTaskRunRunner(f, config.GetTaskRun()).
 				CreateTaskRun().
 				ExpectSuccess().
-				ExpectLogs(config.GetAllExpectedLogs()...).
-				ExpectResults(map[string]string{
-					"name":      config.TaskData.ResultTemplateName,
-					"namespace": config.TaskData.TargetNamespace,
-				})
+				ExpectLogs(config.GetAllExpectedLogs()...)
 
 			results := r.GetResults()
 			resultTemplateName := results["name"]
 			resultTemplateNamespace := results["namespace"]
 
 			newTemplate, err := f.TemplateClient.Templates(resultTemplateNamespace).Get(context.TODO(), resultTemplateName, v1.GetOptions{})
 			Expect(err).ShouldNot(HaveOccurred())
-			Expect(newTemplate).ToNot(Equal(nil), "new template should exists")
+			Expect(newTemplate).ToNot(BeNil(), "new template should exists")
 
 			f.ManageTemplates(newTemplate)
 		},
 			table.Entry("should create template in the same namespace", &testconfigs.CopyTemplateTestConfig{
 				TaskRunTestConfig: testconfigs.TaskRunTestConfig{
 					ServiceAccount: CopyTemplateServiceAccountName,
 				},
-				TaskData: testconfigs.CopyTemplateTaskData{},
+				TaskData: testconfigs.CopyTemplateTaskData{
+					Template: testtemplate.NewCirrosServerTinyTemplate().Build(),
+				},
 			}),
 			table.Entry("should create template in different namespace", &testconfigs.CopyTemplateTestConfig{
 				TaskRunTestConfig: testconfigs.TaskRunTestConfig{
 					ServiceAccount: CopyTemplateServiceAccountName,
 					LimitTestScope: ClusterTestScope,
 				},
 				TaskData: testconfigs.CopyTemplateTaskData{
-					TargetTemplateNamespace: openshiftNamespace,
+					TargetTemplateNamespace: TestTargetNS,
```

Comment:
both source and target namespace default to the test one right? In that case DeployTargetNS should be here to create a template in a different namespace.